client/src/pages/messages/Messages.vue

```diff
@@ -0,0 +1,1020 @@
+                       icon="fas fa-reply"
+                       no-caps
+                       outline
+                       rounded
+                       color="primary"
+                       label="Reply"
+                       @click="replyMsg"/>
+                <q-btn size="sm"
+                       icon-right="fas fa-share"
+                       no-caps
+                       outline
+                       rounded
+                       color="primary"
+                       label="Forward"
+                       @click="forwardMsg"/>
+              </div>
+              <div>
+
+                <q-btn flat size="sm" icon="delete" @click="openDeleteConfirm"/>
+              </div>
+            </div>
+
+          </template>
+
+          <template v-for="(_, slot) of $slots" v-slot:[slot]="scope">
+            <slot :name="slot" v-bind="scope"/>
+          </template>
+        </table-template>
+      </div>
+      <inbox-dialog v-model="showInbox" style="display: flex" :title="dialogTitle" @close="closeDialog">
+        <inbox-form :message="msgToEdit" :is-reply="isReply" @sent="sent"/>
+      </inbox-dialog>
+    </template>
+  </dashboard-layout>
+</template>
+
+<script>
+  import {capitalize, kebabize, singularize} from '@sparkz-community/common-client-lib/src/utils';
+  import DashboardLayout from 'components/dashboards/DashboardLayout';
+  import AccountsFilter from 'pages/messages/components/accounts-filter';
+  import DatePiker from 'pages/messages/components/date-piker';
+  import InboxDialog from 'pages/messages/components/inbox-dialog';
+  import InboxForm from 'pages/messages/components/inbox-form';
+  import {routerMixin, useFindPaginate} from '@sparkz-community/common-client-lib';
+  import TableTemplate
+    from '@sparkz-community/common-client-lib/src/components/common/molecules/tables/TableTemplate.vue';
+  import RandomAvatar from 'components/profile/RandomAvatar/RandomAvatar';
+  import VueGroupAvatar from 'pages/messages/components/VueGroupAvatar/VueGroupAvatar';
+
+  import useMessages from 'stores/services/messages';
+
+  export default {
+    name: 'Messages',
+    components: {
+      VueGroupAvatar,
+      RandomAvatar,
+      TableTemplate,
+      InboxForm,
+      InboxDialog,
+      DatePiker,
+      AccountsFilter,
+      DashboardLayout,
+    },
+    setup() {
+      const messagesStore = useMessages;
+
+      useFindPaginate({
+        model: messagesStore.model,
+        qid: 'messages',
+        skip: 0,
+        limit: 8,
+        query() {
+          // console.log('twaha: ',query);
+          return Object.assign(
+            {},
+            {
+              $sort: {
+                updatedAt: -1,
+                status: 'new',
+              },
+            },
+            this.linkQuery,
+            this.selectedFilterQuery,
+            this.searchBoxQuery,
+          );
+        },
+        params() {
+          return this.params;
+        },
```

Comment:
params and query need to be Computed properties. Also skip, limit, and qid need to be refs.